assets/src/block-editor/TaxonomySelector/index.js

```diff
@@ -8,6 +8,8 @@ export const TaxonomySelector = memo(props => {
 
   return useMemo(() => (
     <FormTokenField
+      __nextHasNoMarginBottom
```

Comment:
On my local, there is a huge gap between the Categories listing and the button to add a new category.


https://github.com/user-attachments/assets/02aa811b-6a9e-4c38-bf07-909e70d68f06

Reply:
I cannot reproduce this, can you please check again? And upgrade to WordPress 6.8.1?